### PR TITLE
enable upgrade

### LINKED ISSUE #3175: Need to add an upgrade test to full-ci test suite.
I'd like for us to add at least one test where the previous OSS released version is installed and then upgraded to the current build. We should then do a some basic use case tests with the upgraded install.

### PR DESCRIPTION
**Pull request by emlin**
Fixes #3175

Re-enable upgrade in vic-machine and enable upgrade test

### CODE REVIEW

File: tests/test-cases/Group11-Upgrade/11-01-Upgrade.robot [hunk 1]
```diff
@@ -1,13 +1,13 @@
...
-# Suite Setup  Install VIC with version to Test Server  5604
-# Suite Teardown  Clean up VIC Appliance And Local Binary
+Suite Setup  Install VIC with version to Test Server  7301
+Suite Teardown  Clean up VIC Appliance And Local Binary
 Default Tags
 
 *** Keywords ***
 Install VIC with version to Test Server
-    [Arguments]  ${version}=5604
+    [Arguments]  ${version}=7301
```
Comment (mdubya66) [inline tests/test-cases/Group11-Upgrade/11-01-Upgrade.robot]: Eventually this will need to be 7315 as that will be our GA version.

Comment (emlin) [inline tests/test-cases/Group11-Upgrade/11-01-Upgrade.robot]: okay

File: tests/test-cases/Group11-Upgrade/11-01-Upgrade.robot [hunk 2]
```diff
@@ -33,69 +33,74 @@ Launch Container
...
 Upgrade VCH with containers
-    Log  Disabled until PR \#2554 is merged  WARN
-    # ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} network create bar
-    # Should Be Equal As Integers  ${rc}  0
-    # Comment  Launch first container on bridge network
-    # ${id1}  ${ip1}=  Launch Container  vch-restart-test1  bridge
-    # ${id2}  ${ip2}=  Launch Container  vch-restart-test2  bridge
+    ${rc}  ${output}=  Run And Return Rc And Output  docker %{VCH-PARAMS} network create bar
+    Should Be Equal As Integers  ${rc}  0
+    Comment  Launch first container on bridge network
```
Comment (hmahmood) [inline tests/test-cases/Group11-Upgrade/11-01-Upgrade.robot]: Does this comment need to be updated? We are launching not one but two containers below.

**Comment by emlin** [comment]
> Not doing a full-ci on this one?

Did a full ci before mike's merge, and it failed only one for the docker rm -f, which is known issue.
So don't run full ci again. I'll rerun the upgrade test locally

**Comment by chengwang86** [comment]
LGTM

**Comment by mdubya66** [comment]
Approved for merging once Mike Hagen says go

**Comment by emlin** [comment]
.

**Pull request merged**